entrypoint

```diff
@@ -133,7 +135,7 @@ if [[ -n "$DB_RESTORE_TARGET" ]]; then
   fi
   # did we get a file?
   if [[ -f "$TMPRESTORE" ]]; then
-    workdir=/tmp/restore.$$
+    workdir="/${TMP_PATH}/restore.$$"
```

Comment:
This would lead to a double `/`, e.g. if `TMP_PATH=/tmp`, it would resolve to `//tmp/restore.$$`. That isn't terrible - in most cases it would be fine - but let's get it correct.